api/helpers/game-states/moves/stalemate-request/validate.js

```diff
@@ -45,11 +56,6 @@ module.exports = {
         throw new Error('game.snackbar.stalemate.previousStalemateRejected');
       }
 
-      // Must not already be considering a stalemate
-      if (currentState.phase === GamePhase.STALEMATE_REQUEST) {
-        throw new Error('game.snackbar.global.alreadyConsideringStalemate');
-      }
-
```

Comment:
Moved this error first and swapped the error key from `global` to `stalemate` to ensure it gets top priority and that it's grouped with the other stalemate validation